### DISCUSSION TITLE
how to setup zluda

### DISCUSSION THREAD

**Discussion by ghost** [vosen/ZLUDA#82] [Q&A] [answered]
hi im finding the setup instructions confusing ive managed to install all prerequisites though c++ compiler wasn't obvious on which one to use so i assume any of the ones available online to install are suitable   

when you say use command line for windows what 1 ? command prompt? python 3? the c++ compiler ? they all have command lines i dont understand where we are meant to set the program up neither can i find a exe for zluda so im assuming it has to be run through a environment of some kind? also is there a script you could paste into power shell or command prompt that would set zluda up automatically or do you have to run all the code 

i did try to research this but couldn't find any videos or favourably step by step processes on how to setup zluda. thanks

**Comment by DGdev91** [comment, marked as answer]
you don't have to do any of that. those instructions are for building it from source, and that's not what you want if you just want to try it.

just do this:

- download  the .zip from here https://github.com/vosen/ZLUDA/releases/tag/v3
- extract it
- open the command prompt
- write <ZLUDA_DIRECTORY>\zluda.exe -- <APPLICATION> <APPLICATION_ARGUMENTS>

for example, let's say you have put it in C:\Zluda\ and you want to open the program "example.exe" wich is in C:\example\
then you should write

C:\Zluda\zluda.exe -- example.exe

that's it!

**Reply by ghost** [reply]
oh i see well this cleared up all confusion though the zluda exe runs and then the prompt crashes i followed your instructions am i missing something? should i delete the old version which is zluda master?

**Reply by DGdev91** [reply]
yes, you don't need the source folder, just download the v3 zip and unpackage it somewhere.

if it crashes, there can be many reason. maybe your card is not supported or the program you were trying is too complex yet for running under zluda.
Also, as said in the instructions, if you also have an integrated gpu you should set the HIP_VISIBLE_DEVICES=1 enviroment variable

**Reply by ghost** [reply]
Oh i understand thank you very much for helping me so far hope this info helps 

PC specs
Cpu Ryzen 7 5800x 3D
Motherboard Asus tuf 570x pro
Ram 32 GB ddr 4
Gpu radeon 7900xtx
Psu 1000 watt

Cpu doesn't have integrated graphics according to AMD site

Zip has been uncompressed/unpackaged

Zluda exe crashes before being able to run any programs

**Reply by tms369** [reply]
With your specs it should work. 

eg: C:\Zluda>zluda.exe -- C:\blender\blender.exe

don't forget the space after --.

**Reply by ghost** [reply]
where do you want me to put that ?

**Reply by tms369** [reply]
run cmd from the zluda directory, i.e. just type cmd on the address bar from that folder.

**Reply by ghost** [reply]
how do you run a cmd in a file explorer folder?

**Reply by ghost** [reply]
![needdddannoyinghelpppp](https://github.com/vosen/ZLUDA/assets/159965833/7a6f02bf-f394-43a6-8aa9-d57172613f8a)
it didn't do anything ?

**Reply by tms369** [reply]
you already at the directory. Just type 'zluda.exe -- (app folder)/app.exe'.

**Reply by ghost** [reply]
it says (this app cant run on your PC)

**Reply by tms369** [reply]
![Untitled1](https://github.com/vosen/ZLUDA/assets/108114632/fc0db385-9154-46fe-8e67-91d025d70a37)
Maybe try another application.

**Reply by ghost** [reply]
i did bledner-launcher.exe -- (app folder)/app.exe and it opened fine
